kuma/static/stylus/base/fonts/compat-icons.styl

```diff
@@ -1,10 +1,10 @@
 @font-face {
     font-family: 'compat-icons';
-    src:url($path-to-fonts + 'compat-icons.eot?-ehmwve');
-    src:url($path-to-fonts + 'compat-icons.eot?#iefix-ehmwve') format('embedded-opentype'),
-        url($path-to-fonts + 'compat-icons.woff?-ehmwve') format('woff'),
-        url($path-to-fonts + 'compat-icons.ttf?-ehmwve') format('truetype'),
-        url($path-to-fonts + 'compat-icons.svg?-ehmwve#compat-icons') format('svg');
```

Comment:
Given that the SVG source is removed, can we also remove the *kuma/static/fonts/compat-icons.svg* file?